crates/autopilot/src/run_loop.rs

```diff
@@ -316,6 +322,7 @@ impl RunLoop {
                 Metrics::fee_policies_store_error();
                 tracing::warn!(?err, "failed to save fee policies");
             }
+            Metrics::competition_stored(start.elapsed());
```

Comment:
Maybe we can move everything between the first start and this into a prost_processing method to keep single_run a bit more readable?

Reply:
You mean the actual code, not the metrics, right? I will open a separate PR since it would be easier to read the actual changes.

Reply:
https://github.com/cowprotocol/services/pull/2895